src/loader/bgw_launcher.c

```diff
@@ -35,6 +35,8 @@
 #include "bgw_message_queue.h"
 #include "bgw_launcher.h"
 
+#include <tcop/tcopprot.h>
```

Comment:
I don't think this is needed anymore, it was just there for debugging

Reply:
This is needed for die(), added clarification and put it in a better place in the headers